network/vpc/concepts.mdx

```diff
@@ -13,40 +13,99 @@ dates:
   posted: 2023-02-06
 ---
 
-<Message type="note">
+<Concept opened>
 
-  This page concerns the **VPC** / **regional Private Networks** product currently in [Public Beta](https://www.scaleway.com/en/betas/#regional-private-network). If you are looking for documentation about Public Gateways, see the [relocated concepts page](/network/public-gateways/concepts/) for that product.
+## CIDR block
 
-</Message>
+A **C**lassless **I**nter-**D**omain **R**outing (CIDR) block represents a range of IP addresses. They all share the same network prefix. An example of an [IPv4](#ipv4) CIDR block is `172.16.16.0/22`, while an [IPv6](#ipv6) CIDR block looks like `2002::1234:abcd:ffff:c0a8:101/64`.
```

Comment:
I recommend using a CIDR similar to the following one for IPv6: fd12:3456:789a:1::/64
Source: https://en.wikipedia.org/wiki/Unique_local_address#Example

Reply:
Done